wren-ai-service/src/pipelines/generation/sql_regeneration.py

```diff
@@ -109,20 +155,24 @@ async def sql_regeneration_post_process(
 ## End of Pipeline
 
 
-class Generation(BasicPipeline):
+class SQLRegeneration(BasicPipeline):
     def __init__(
         self,
         llm_provider: LLMProvider,
         engine: Engine,
     ):
-        self.sql_regeneration_preprocesser = SQLRegenerationRreprocesser()
-        self.sql_regeneration_prompt_builder = PromptBuilder(
-            template=sql_regeneration_user_prompt_template
-        )
-        self.sql_regeneration_generator = llm_provider.get_generator(
-            system_prompt=sql_regeneration_system_prompt
-        )
-        self.sql_regeneration_post_processor = GenerationPostProcessor(engine=engine)
+        self._components = {
+            "sql_regeneration_preprocesser": SQLRegenerationRreprocesser(),
+            "sql_regeneration_prompt_builder": PromptBuilder(
+                template=sql_regeneration_user_prompt_template
+            ),
+            "sql_regeneration_generator": llm_provider.get_generator(
+                system_prompt=sql_regeneration_system_prompt
+            ),
+            "sql_regeneration_post_processor": SQLBreakdownGenerationPostProcessor(
```

Comment:
I think we could consider removing the prefix `sql_regeneration` to be more concise.

Reply:
fixed